### DISCUSSION TITLE
Discounts >> Add condition for Payment Method

### DISCUSSION THREAD

**Discussion by wmdhosting** [craftcms/commerce#3722] [Ideas] [answered]
There is option for shipping methods, but will be great if you can add gateways too.. **Payment method,** 
As there are often cases to get discount if customer choose some kind of Payment method

![2024-10-14_18-46-02](https://github.com/user-attachments/assets/ae67fe29-761a-4018-8750-c83c1cc745c5)

**Comment by lukeholder** [comment]
This will be included in Commerce 5.3, so stay tuned for that

github.com/craftcms/commerce/commit/1ad4c421414ab7456cc81e943eb85eeadaad5401

**Comment by nfourtythree** [comment, marked as answer]
Commerce 5.3 has now been released with this included.

Thanks!
